docs/source/release/v6.12.0/index.rst

```diff
@@ -4,19 +4,28 @@
 Mantid 6.12.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../../../images/CreateMonteCarloWorkspace_spectrum.png
```

Comment:
It is not clear what this image relates to. Perhaps below where we highlight the specific change linked to this image we say something like see image above?

Reply:
done